test/src/xquery/modules/sql/suite.xql

```diff
@@ -0,0 +1,3 @@
+xquery version "3.0";
+
+<empty/>
```

Comment:
That was needed because I used the TestRunner base class - I originally planned to write the test using XQUnit test. I will change it.